packages/rpc/src/tezos/models/TezosBlockHeader.ts

```diff
@@ -0,0 +1,17 @@
+import BigNumber from 'bignumber.js';
+import { Transform } from 'class-transformer';
+
+export class TezosBlockHeader {
+    @Transform(value => new BigNumber(value), { toClassOnly: true })
+    level: BigNumber;
+    proto: number;
+    predecessor: string;
+    timestamp: string;
+    validation_pass: number;
```

Comment:
We use camelcase in all properties and methods. Can you change it to camelcase and use `@Expose({ name: "validation_pass" })`?

Reply:
Got it!.. Making the change now.